### PR TITLE
new feature

### PR DESCRIPTION
**Pull request by BloodyPikachu**
### Description of Your Pull Request
What does your pull request specifically does by adding/modifying or removing code?
It modifies msg.guild to guild. Adds cool features to serverinfo(guild).js including members talked.
### Changes Proposed and or Files Changed
- Example Change
It will track how many members have talked in a server since last bot/shard update.
### Label Your Pull Request
- [ Yes] This PR modifies existing code to enhance the bot
- [Yes ] This PR removes/renames methods or properties
- [Yes ] This PR adds methods, features, commands or properties

### CODE REVIEW

**Comment by AdityaTD** [comment]
The code needs to be more efficiently done, discussed this further on Discord.

File: commands/Utilities/guild.js [hunk 1]
```diff
@@ -17,21 +17,61 @@ module.exports = class extends Command {
     }
 
     async run(msg) {
+      const guild = msg.guild
```
Comment (MrJacz) [inline commands/Utilities/guild.js]: should destructure but this is useless.

File: commands/Utilities/guild.js [hunk 2]
```diff
@@ -17,21 +17,61 @@ module.exports = class extends Command {
...
+      const server = {
```
Comment (MrJacz) [inline commands/Utilities/guild.js]: Useless object

File: commands/Utilities/guild.js [hunk 3]
```diff
@@ -17,21 +17,61 @@ module.exports = class extends Command {
...
+        name: guild.name,
+        id: guild.id,
+        createdAt: guild.createdAt,
+        image: guild.iconURL(),
+        botJoin: guild.joinedAt,
+        region: guild.region.toUpperCase(),
+        owner: `${guild.owner.displayName}#${guild.owner.user.discriminator}`,
```
Comment (MrJacz) [inline commands/Utilities/guild.js]: owner can be null.

File: commands/Utilities/guild.js [hunk 4]
```diff
@@ -17,21 +17,61 @@ module.exports = class extends Command {
...
+        filter: guild.explicitContentFilter,
+        roles: guild.roles.array().length,
```
Comment (MrJacz) [inline commands/Utilities/guild.js]: `guild.roles.size`

File: commands/Utilities/guild.js [hunk 5]
```diff
@@ -17,21 +17,61 @@ module.exports = class extends Command {
...
+        members: guild.memberCount,
+        bots: guild.members.filter((member) => { return member.user.bot }).array().length,
```
Comment (MrJacz) [inline commands/Utilities/guild.js]: `guild.members.filter(member => member.user.bot).size`
this wont work correctly anyway, not all members are cached.

File: commands/Utilities/guild.js [hunk 6]
```diff
@@ -17,21 +17,61 @@ module.exports = class extends Command {
...
+        activity: guild.members.filter((member) => {
```
Comment (MrJacz) [inline commands/Utilities/guild.js]: this wont work like you think it will, dont use .array()

File: commands/Utilities/guild.js [hunk 7]
```diff
@@ -17,21 +17,61 @@ module.exports = class extends Command {
...
+          if(member.lastMessage) {
+            return member.lastMessage.createdTimestamp
+          }
+        }).array(),
+        daily: guild.members.filter((member) => {
```
Comment (MrJacz) [inline commands/Utilities/guild.js]: same here

File: commands/Utilities/guild.js [hunk 8]
```diff
@@ -17,21 +17,61 @@ module.exports = class extends Command {
...
+          if(member.lastMessage) {
+            if(new Date() - member.lastMessage.createdTimestamp < 86400000) {
+              return member.lastMessage.createdTimestamp
+            }
+          }
+        }).array(),
+        weekly: guild.members.filter((member) => {
```
Comment (MrJacz) [inline commands/Utilities/guild.js]: same

File: commands/Utilities/guild.js [hunk 9]
```diff
@@ -17,21 +17,61 @@ module.exports = class extends Command {
...
+          if(member.lastMessage) {
+            if(new Date() - member.lastMessage.createdTimestamp < (86400000 * 7) && new Date() - member.lastMessage.createdTimestamp > 86400000) {
+              return member.lastMessage.createdTimestamp
+            }
+          }
+        }).array(),
+        monthly: guild.members.filter((member) => {
```
Comment (MrJacz) [inline commands/Utilities/guild.js]: same here

**Pull request closed without merging**